.gitignore

```diff
@@ -6,5 +6,9 @@ node_modules
 .env
 .env.*
 !.env.example
+pnpm-lock.yaml
+.editorconfig
 vite.config.js.timestamp-*
 vite.config.ts.timestamp-*
+
+/.idea
```

Comment:
FYI you can create a global gitignore (in `~/.config/git/ignore`) for your machine to ignore all of your editors' stuff. :)

Reply:
Didn't know that, thanks would do that and update the PR